scripts/host-testbench.sh

```diff
@@ -44,6 +44,55 @@ function srcsize() {
   echo "$OUTPUT_SIZE"
 }
 
+function test_comp() {
```

Comment:
The `function` keyword is not required and less portable.

Reply:
ditto